openshift/ci-operator/update-ci.sh

```diff
@@ -38,9 +38,8 @@ stage "Generating CI config files"
 CONFIG=$CONFIGDIR/openshift-knative-eventing-release-$VERSION
 PERIODIC_CONFIG=$PERIODIC_CONFIGDIR/openshift-knative-eventing-release-$VERSION-periodics.yaml
 CURDIR=$(dirname $0)
-$CURDIR/generate-ci-config.sh knative-$VERSION 4.8 true false > ${CONFIG}__48.yaml
 $CURDIR/generate-ci-config.sh knative-$VERSION 4.9 true false > ${CONFIG}__49.yaml
-$CURDIR/generate-ci-config.sh knative-$VERSION 4.10 false true > ${CONFIG}__410.yaml
+$CURDIR/generate-ci-config.sh knative-$VERSION 4.10 true false > ${CONFIG}__410.yaml
```

Comment:
I'd do

```
$CURDIR/generate-ci-config.sh knative-$VERSION 4.9 true false > ${CONFIG}__49.yaml
$CURDIR/generate-ci-config.sh knative-$VERSION 4.10 false true > ${CONFIG}__410.yaml
```
First arg is "disable image promotion"
Second is "run periodics"

If the first arg is TRUE for all jobs: we have no images promoted. (we can only have on job with promotion, all others are needing the `true` for disabling that)

And I'd suggest that for 4.10 we do run the periodic